src/test/groovy/org/ods/doc/gen/leva/doc/fixture/LevaDocTestValidator.groovy

```diff
@@ -23,7 +28,12 @@ class LevaDocTestValidator {
     boolean validatePDF(String buildId) {
         unzipGeneratedArtifact(buildId)
         if (GENERATE_EXPECTED_PDF_FILES) {
-            copyDocWhenRecording(buildId)
+            boolean comparisonResult = compareFiles(buildId)
```

Comment:
Please remove all this part as it will add a lot of time to the test, when if this is GENERATE_EXPECTED_PDF_FILES=true, we expect to copy the results.

Reply:
Please take a look at the whole change, not just two lines.
We are updating the docs, but only if we need to.
Pdfs source code can change while the visual result remains the same.
Please comment again if you want us to remove it.

Reply:
Please, don't put this code, this will add a lot of time and not needed